cmd/protecodeExecuteScan_generated.go

```diff
@@ -0,0 +1,299 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/SAP/jenkins-library/pkg/config"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/piperenv"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/spf13/cobra"
+)
+
+type protecodeExecuteScanOptions struct {
+	ProtecodeExcludeCVEs                 string `json:"protecodeExcludeCVEs,omitempty"`
+	ProtecodeFailOnSevereVulnerabilities bool   `json:"protecodeFailOnSevereVulnerabilities,omitempty"`
+	ScanImage                            string `json:"scanImage,omitempty"`
+	DockerRegistryURL                    string `json:"dockerRegistryUrl,omitempty"`
+	CleanupMode                          string `json:"cleanupMode,omitempty"`
+	FilePath                             string `json:"filePath,omitempty"`
+	IncludeLayers                        bool   `json:"includeLayers,omitempty"`
+	AddSideBarLink                       bool   `json:"addSideBarLink,omitempty"`
+	Verbose                              bool   `json:"verbose,omitempty"`
```

Comment:
we probably don't need this since the areas I saw where it was used are handled by the logging automatically.
Maybe I overlooked something though.

Reply:
`Verbose` can be removed from the metadata. We integrated this is a central part of the config resolution. Hence documentation still needs to be enhanced.